scripts/revert_changes_from_user.pl

```diff
@@ -23,20 +23,19 @@
 use Modern::Perl '2012';
 use utf8;
 
-my $reverted_user_id = "equadis";
-
 my $usage = <<TXT
 revert_changes_from_user.pl
 
 This script will revert products to the most recent version before a given user changed the product.
 Products that did not exist before will be deleted.
 All changes done after the first edit of the user will also be deleted, even if done by other users.
 
+You may limit to a series of codes
+
 Usage:
 
-update_all_products.pl --pretend
+revert_changes_from_user.pl --userid user_id --codes codes --pretend
```

Comment:
@alexgarel : the --codes parameter is not coded yet I believe?

Reply:
Right, we should remove it ! (I saw it was too much time to implement it).